src/components/try-hcp-callout/components/product-icon-heading/index.tsx

```diff
@@ -1,44 +1,29 @@
-import InlineSvg from '@hashicorp/react-inline-svg'
-import { useId } from '@react-aria/utils'
-import hcpLogo from '!!raw-loader!@hashicorp/mktg-logos/product/hcp/primary/black.svg'
+import classNames from 'classnames'
 import ProductIcon from 'components/product-icon'
 import { ProductSlug } from 'types/products'
 import s from './product-icon-heading.module.css'
 
 export interface ProductIconHeadingProps {
-	productSlug: Exclude<ProductSlug, 'sentinel'>
+	productSlug: Exclude<ProductSlug, 'sentinel' | 'hcp'>
 	headingText: string
+	size?: 'small' | 'medium'
 }
 
 /**
  * A card heading component, for use in HCP callout cards.
- *
- * Note that when "hcp" is provided as the product slug, we render
- * the full HCP logo, and we render the provided headingText as visually hidden.
  */
 export function ProductIconHeading({
 	productSlug,
 	headingText,
+	size = 'medium',
 }: ProductIconHeadingProps) {
-	const uniqueId = useId()
-
-	if (productSlug === 'hcp') {
```

Comment:
Have split out the `"hcp"` case as it's only used in `TryHcpCallout`. This prevents `TryHcpCalloutCompact` from loading in the full HCP logo, when it won't be used.

Reply:
nice, I like it!